taxi/urls.py

```diff
@@ -16,12 +23,30 @@
         ManufacturerListView.as_view(),
         name="manufacturer-list",
     ),
-    path("cars/", CarListView.as_view(), name="car-list"),
-    path("cars/<int:pk>/", CarDetailView.as_view(), name="car-detail"),
+    path(
+        "manufacturer/create",
+        ManufacturerCreateView.as_view(),
+        name="manufacturer-create"
+    ),
+    path(
+        "manufacturer/<int:pk>/update",
+        ManufacturerUpdateView.as_view(),
+        name="manufacturer-update"
+    ),
+    path(
+        "manufacturer/<int:pk>/delete",
```

Comment:
Consider adding a trailing slash to the 'manufacturer/<int:pk>/delete' path for consistency with Django's URL conventions.